src/pixie/blends.nim

```diff
@@ -51,17 +56,21 @@ proc blendNormalPremultiplied*(backdrop, source: ColorRGBA): ColorRGBA {.inline.
   result.r = source.r + ((backdrop.r.uint32 * k) div 255).uint8
   result.g = source.g + ((backdrop.g.uint32 * k) div 255).uint8
   result.b = source.b + ((backdrop.b.uint32 * k) div 255).uint8
-  result.a = source.a + ((backdrop.a.uint32 * k) div 255).uint8
+  result.a = blendAlpha(backdrop.a, source.a)
+
+proc blenderPremultiplied*(blendMode: BlendMode): Blender =
+  case blendMode:
+  of bmNormal: blendNormalPremultiplied
+  else:
+    raise newException(PixieError, "No premultiplied blender for " & $blendMode)
```

Comment:
I though we had blends for others (event they they just convert back and forth)?